cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/BulkSparkConf.java

```diff
@@ -146,17 +148,16 @@ public class BulkSparkConf implements Serializable
     protected boolean useOpenSsl;
     protected int ringRetryCount;
     // create sidecarInstances from sidecarInstancesValue and effectiveSidecarPort
-    private final String sidecarInstancesValue;
+    private final String sidecarContactPointsCsv;
```

Comment:
Not a huge fan of this variable name, can we use `sidecarContactPointsValue` instead?

Reply:
ok.